### PR TITLE
BIS docs refactoring

### PR DESCRIPTION
**Pull request by diegoalexdiaz**
Refactoring of BIS documentation, including the intro-docs as well as the schema-reference sections.

### CODE REVIEW

File: docs/bis/index.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
 BIS is a “conceptual schema” that expresses taxonomy, data structure, and relationships for modeling real-world Entities. It is written using Bentley’s open [“EC” schema language](../ec/ec-schema.md) for Entity-Relationship modeling.
 
 ## Open and Extensible
 
-BIS is an “open” and extensible family of schemas. It is modularized into [Domains](./intro/glossary.md#Domain). The “BIS Core” Domain expresses the fundamental modeling concepts. Each Domain is expressed as a separate [ECSchema](../ec/ec-schema.md). Anyone can author a new Domain schema by following the rules and guidelines in this documentation. Users can also extend Domain schemas by adding custom classes, properties, and relationships.
+BIS is an “open” and extensible family of schemas. It is modularized into [Domains](./fundamentals/glossary.md#Domain). The “BIS Core” Domain expresses the fundamental modeling concepts. Each Domain is expressed as a separate [ECSchema](../ec/ec-schema.md). Anyone can author a new Domain schema by following the rules and guidelines in this documentation. Users can also extend Domain schemas by adding custom classes, properties, and relationships.
```
[the remark this answers is not in the document]
Comment (diegoalexdiaz) [reply docs/bis/index.md:9]: Added a link from the glossary, after renaming "Schema Reference" to "Domain schemas" as suggested.

File: docs/bis/guide/data-organization/information-hierarchy.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+Each of these mechanism is intended to be used in specific circumstances which are explained in this chapter.
+
+### The Subject Hierarchy Organizes Models
+
+The RepositoryModel contains a hierarchy of `Subject` Elements organized into a parent-child hierarchy. All Models (other than the RepositoryModel itself) are organized under these `Subject`s (via InformationPartition)
+
+### Model Contains Elements
+
+A Model is a *container* for Elements.
```
Comment (pmconne) [inline docs/bis/guide/data-organization/information-hierarchy.md:29]: Sentences on separate lines with no blank lines in between render as a single paragraph. Was that the intent or were you intending to make a list like under "Hierarchy Constructs"?

Comment (diegoalexdiaz) [reply docs/bis/guide/data-organization/information-hierarchy.md:29]: These particular sentences were not changed wrt their original writing currently in production. No intention to make a list so it is fine that they render as a single paragraph.

**Review by pmconne — COMMENTED** [review]
Why is some documentation in .pptx files? How are those presented on the docs site? They render very poorly in LibreOffice.

**Review by ColinKerr — APPROVED** [review]

**Pull request merged**